MomsSpaghetti/Pages/Index.cshtml.cs

```diff
@@ -75,7 +74,7 @@ public async Task<IActionResult> OnGet()
 
                     {
 
-                        recipeDetails = await response.Content.ReadAsStringAsync();
+                        string recipeDetails = await response.Content.ReadAsStringAsync();
```

Comment:
Created  a string variable closer to the place where it is being used .
Source: https://www.geeksforgeeks.org/c-sharp-coding-standards/